src/components/experiments/ExperimentTable.jsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { List } from "immutable";
+
+export class ExperimentTable extends React.Component {
+  render() {
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Start Date</th>
+            <th>End Date</th>
+            <th>Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          {this.props.filteredExperiments.map(function(item, key) {
+            return (
+              <tr key={key}>
+                <td>{item.get("name")}</td>
+                <td>{new Date(item.get("start_date")).toGMTString()}</td>
```

Comment:
I think all the times (not the dates) that are like 00:00:00 are probably unnecessary, maybe just only show the dates?